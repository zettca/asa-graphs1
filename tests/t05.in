20 50
19
19 6
15 16
15 12
15 8
15 5
15 9
15 3
18 14
18 11
18 13
18 16
18 12
18 8
1 4
1 10
1 11
1 20
1 8
1 7
2 6
2 20
2 5
2 9
6 4
6 17
4 11
4 20
4 16
4 12
14 10
14 11
14 20
14 16
14 8
14 9
14 3
10 11
10 5
10 3
11 13
11 16
11 8
17 8
17 5
17 7
20 12
20 9
13 8
13 3
8 7
