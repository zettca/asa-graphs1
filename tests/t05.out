5
1
3
9
5
1
